tests/robot-tests/tests/public_api/public_api_major_auto_changes.robot

```diff
@@ -0,0 +1,253 @@
+    user checks summary list contains    Indicators    Enrolments    parent=id:draft-version-summary
+    user checks summary list contains    Indicators    more indicators    parent=id:draft-version-summary
+    user checks summary list contains    Filters    Academy type    parent=id:draft-version-summary
+    user checks summary list does not contain    Filters    School type    parent=id:draft-version-summary
+    user checks summary list contains    Actions    Remove draft version    parent=id:draft-version-summary
+
+Validate that location and filter mappings are complete
+    user waits until h3 is visible    Draft version tasks
+    user waits until parent contains element    testid:map-locations-task    link:Map locations
+    user waits until parent contains element    id:map-locations-task-status    text:Complete
+    user waits until parent contains element    testid:map-filters-task    link:Map filters
+    user waits until parent contains element    id:map-filters-task-status    text:Complete
+
+Confirm finalization of this API data set version
+    user clicks button    Finalise this data set version
+    user waits for caches to expire
+    user waits until h2 is visible    Mappings finalised
+    user waits until page contains    Draft API data set version is ready to be published
+
+User navigates to 'changelog and guidance notes' page and update relevant details in it
+    user clicks link by index    View changelog and guidance notes    1
+    user waits until page contains    API data set changelog
+
+    user enters text into element    css:textarea[id="guidanceNotesForm-notes"]
+    ...    Content for the public guidance notes
+    user clicks button    Save public guidance notes
+
+    user waits until page contains    Content for the public guidance notes
+    user clicks link    Back to API data set details
+
+User clicks on 'View preview token log' link inside the 'Draft version details' section
+    user clicks link by index    View changelog and guidance notes    2
+
+Validate the contents in the 'API dataset changelog' page.
+    user waits until page contains    API data set changelog
+
+    user waits until page contains    Content for the public guidance notes
+    user clicks link    Back to API data set details
+
+Add headline text block to Content page for the second release
+    user navigates to content page    ${PUBLICATION_NAME}
+    user adds headlines text block
+    user adds content to headlines text block    Headline text block text
+
+Approve second release
+    user clicks link    Sign off
+    user approves release for immediate publication
+
+Verify newly published release is on Find Statistics page
+    user checks publication is on find statistics page    ${PUBLICATION_NAME}
+
+User navigates to data catalogue page
+    user navigates to data catalogue page on public frontend
+
+Search with 2nd API dataset
```

Comment:
Should this be `Search for API data set`? We didn't upload a second API data set (we only updated a single one).

Reply:
Again, some inherited wording here from the original "resolve mappings" test suite.  It's referring to the next data set *version* e.g. the second subject uploaded and used for this data set.  I've updated the terminology in this and the original suite.

Reply:
Just to clarify, both the *minor* and *major* test suites create an initial data set version and then create a newer version of that same data set.

So in both suites I've renamed "1st data set" to "initial data set version" and "2nd data set" to "new data set version".